app/controllers/oauths_controller.rb

```diff
@@ -40,9 +40,9 @@ def destroy
 
   def link_account(provider)
     if @user = add_provider_to_user(provider)
-      flash[:notice] = "You have successfully linked your GitHub account."
+      flash[:notice] = t('auth.linked_successfully', provider: provider.titleize)
     else
-      flash[:alert] = "There was a problem linking your GitHub account."
+      flash[:alert] = t('auth.link_problem', provider: provider.titleize)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.